src/ContentValidationListener.php

```diff
@@ -9,13 +9,15 @@
 use Zend\EventManager\EventManagerInterface;
 use Zend\EventManager\ListenerAggregateInterface;
 use Zend\Http\Request as HttpRequest;
-use Zend\InputFilter\CollectionInputFilter;
```

Comment:
This actually needs to be in there; current develop branch references the class. I can re-instate it on merge, though.